src/main/java/lotto/service/LottoService.java

```diff
@@ -0,0 +1,28 @@
+package lotto.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import lotto.domain.Lotto;
+import lotto.domain.RandomNumber;
+
+public class LottoService {
+    private static Integer DIVIDED_AMOUNT = 1000;
+
+    public int getPurchaseCount(int purchaseAmount) {
+        return purchaseAmount / DIVIDED_AMOUNT;
+    }
+
+    public List<Lotto> makeLottos(int purchaseCount) {
+        List<Lotto> lottos = new ArrayList<>();
+        for (int i = 0; i < purchaseCount; i++) {
+            Lotto lotto = createLotto();
+            lottos.add(lotto);
+        }
+        return lottos;
```

Comment:
stream으로 표현하면 훨씬 간결할 것 같아요!

```suggestion
        return IntStream.range(0, purchaseCount)
                    .mapToObj(i -> createLotto())
                    .collect(Collectors.toList());
```